azure-quantum/README.md

```diff
@@ -79,7 +79,7 @@ result = job.get_results()
 
 ## Contributing ##
 
-For details on contributing to this repository, see the [contributing guide](../CONTRIBUTING.md).
+For details on contributing to this package, see the [contributing guide](https://github.com/microsoft/qdk-python/blob/main/CONTRIBUTING.md).
```

Comment:
I suggest updating the text so that it appropriate for both a repository on github and a package on PyPi.

Reply:
The README is only for the azure-quantum package, not for the entire repo, so I think it is ok to focus on just package.